### PR TITLE
Updated presence for Plex to work with 4.55.0

### PR DESCRIPTION
**Pull request by whallin**
Updated DOM selectors for the Plex presences for the [latest Plex web build as of April 11th, 2021](https://forums.plex.tv/t/plex-web/20528/351).

**Done the following:**
- Updated DOM selectors
- Added Swedish translation

### CODE REVIEW

**Comment by EncryptedDev** [comment]
Please show proof of the changes working & bump the version

**Comment by whallin** [comment]
> Please show proof of the changes working & bump the version
Alrighty. Here are some screenshots from the version in this PR.

**Screenshot of: Viewing library**

Screenshot of: Viewing library

**Screenshot of: Viewing movie/TV show/VOD information**

Screenshot of: Viewing movie/TV show/VOD information

**Screenshot of: Watching movie**

Screenshot of: Watching movie

**Screenshot of: Watching TV show**

Screenshot of: Watching TV show

File: websites/P/Plex/dist/metadata.json [hunk 1]
```diff
@@ -3,14 +3,20 @@
     "name": "Bas950",
     "id": "241278257335500811"
   },
+  "contributors": [
+    {
+      "name": "William",
+      "id": "421979708755017728"
+    }
+  ],
```
Comment (ririxidev) [inline websites/P/Plex/dist/metadata.json:11]: Changes what you did don't qualify you to add yourself as contributor

**Review by ririxidev — CHANGES_REQUESTED** [review]
.

**Review by ririxidev — APPROVED** [review]
that's okay rn!

**Review by StrikerFRFX — APPROVED** [review]
lgtm as well

**Pull request merged**